src/main/java/io/github/thebusybiscuit/slimefun4/core/guide/options/SlimefunGuideSettings.java

```diff
@@ -249,4 +250,25 @@ public static boolean hasFireworksEnabled(@Nonnull Player p) {
         return true;
     }
 
+    /**
+     * This method checks if the given {@link Player} has enabled the {@link LearningAnimationOption}
+     * in their {@link SlimefunGuide}.
+     * If they enabled this setting, they will see messages in chat about the progress of their {@link Research}.
+     *
+     * @param p The {@link Player}
+     *
+     * @return Whether this {@link Player} wants to info messages in chat when unlocking a {@link Research}
+     */
+    public static boolean hasLearningAnimationEnabled(@Nonnull Player p) {
+        for (SlimefunGuideOption<?> option : options) {
+            if (option instanceof LearningAnimationOption) {
+                LearningAnimationOption learningAnimation = (LearningAnimationOption) option;
+                ItemStack guide = SlimefunGuide.getItem(SlimefunGuideMode.SURVIVAL_MODE);
+                return learningAnimation.getSelectedOption(p, guide).orElse(true);
+            }
+        }
+
+        return true;
```

Comment:
This method is very similar to the method above, therefore I think a private static helper method might be appropriate here to reduce this redundancy.

Reply:
I added a generic one which might seem a little overengineered but I think it will might be useful in the future when there is an option which has a different value than boolean